### ISSUE TITLE
ESM build contains keyword `require` crash web app

### ISSUE THREAD

**Issue by joshuajiangdev** [okta/okta-auth-js#1073]
Hi, I'm getting `ReferenceError: require is not defined` from my web at the line `ul=require("js-cookie")`. My vite build is using build targets `modules`, which according to this libs' `package.json`:
```
  "main": "cjs/index.js",
  "module": "esm/index.js",
  "browser": "dist/okta-auth-js.umd.js",
```

is going to load package from `"esm/index.js"`, so I checked that file and could find the code `var Cookies = require('js-cookie');` there. Curious if this is intended or a mistake to include keyword `require` keyword in an esm build.

**Comment by oleksandrpravosudko-okta** [comment]
Thanks for report @joshuajiangdev.

This issue is likely to be addressed in the the next major releases. Until then, you could either `resolve` umd bundle or use helper [plugin](https://github.com/vitejs/vite/issues/3409#issuecomment-860017996).

**Comment by tejasdharamshi** [comment]
When would the next release?

**Comment by jonrimmer** [comment]
@oleksandrpravosudko-okta 1) Resolving to a umd bundle is not possible as the location on disk may differ between environments when npm workspaces. 2) The plugin you linked only operates in `dev` mode, meaning production bundles are still broken.

Do you have any other suggestions? This is blocking our integration of Okta.

**Comment by shuowu** [comment]
Fixed in [v6.2](https://github.com/okta/okta-auth-js/releases/tag/okta-auth-js-6.2.0).
